### ISSUE TITLE
[Feature]: Add condition "user" to conditional chips card

### ISSUE THREAD

**Issue by FaBRiK74** [piitaya/lovelace-mushroom#601]
### Is your feature request related to a problem?

I'd like to add a condition possibility to conditional chips card only show if a given user or user group (like admins) uses the group.

### Describe the solution you'd like

Add condition: Users or Group

**Comment by FaBRiK74** [comment]
Unfortunately it doesn't work. My username is fabrik. When I use the code below, it doesn't show the available HACS update. If I remove the template lines, it shows for everyone:
`  - type: conditional
    conditions:
      - entity: sensor.hacs
        state_not: '0'
      - template: '{{ user == ''fabrik'' }}'
    chip:
      type: entity
      entity: update.hacs_update
      tap_action:
        action: navigate
        navigation_path: /hacs
      name: HACS
      content_info: name
      icon_color: blue
      icon: hacs:hacs`

**Comment by FaBRiK74** [comment]
`  - type: conditional
    conditions:
      - entity: sensor.hacs
        state_not: '0'
      - template: '{{ user == ''fabrik'' }}'
    chip:
      type: entity
      entity: update.hacs_update
      tap_action:
        action: navigate
        navigation_path: /hacs
      name: HACS
      content_info: name
      icon_color: blue
      icon: hacs:hacs`

**Comment by Gnol86** [comment]
> Unfortunately it doesn't work.

It was a proposal to improve your request.

**Comment by FaBRiK74** [comment]
oh, my bad. Sorry :D

**Comment by andker87** [comment]
it would be great to have a "visibility" option for each mushroom card, not just for the chips.
it might be something like the visibility in the lovelace panels, but related to the single cards.

**Comment by piitaya** [comment]
> it would be great to have a "visibility" option for each mushroom card, not just for the chips.
> it might be something like the visibility in the lovelace panels, but related to the single cards.

There is an official conditional card for that : https://www.home-assistant.io/dashboards/conditional/

**Comment by AlessandroTischer** [comment]
I had the same need. I kinda solved with this:
```type: custom:state-switch
theme: mashrooms
entity: user
default: default
states:
  Alessandro:
    type: custom:mushroom-template-card
    primary: Privato
    secondary: ''
    icon: mdi:shield-account
    layout: vertical
    icon_color: none
    fill_container: true
    badge_icon: ''
    tap_action:
      action: navigate
      navigation_path: /lovelace-mobile/privato
    hold_action:
      action: none
    double_tap_action:
      action: none
  default:
    type: markdown content:> Altro Utente
``` 

It works ok, but sometimes it doesn't show the card even if the user is correct. 
Note: since I didn't want any card to show to other users, I removed "default : default", so it never goes into that branch. Instead, removing the default branch itself makes the card not working.

**Comment by mcolde** [comment]
> ing the default branch itself

Unfortunately the State-switch addon does not work for me.
The "visibility" atribute would be awesome. Could work for cards the same way is in panels.
![image](https://user-images.githubusercontent.com/96849333/219005417-d1001409-aae5-49e5-85fb-001be109f2cf.png)

**Comment by piitaya** [comment]
Fixed with https://github.com/piitaya/lovelace-mushroom/pull/1269. It will be available with `2023.11` Home Assistant release.
